gen/dcos-config.yaml

```diff
@@ -2686,3 +2686,52 @@ package:
       DCOS_UI_UPDATE_DIST_PATH=/var/lib/dcos/dcos-ui-update-service/dist
       DCOS_UI_UPDATE_DIST_LINK=/var/lib/dcos/dcos-ui-update-service/dist/ui
       DCOS_UI_UPDATE_STAGE_LINK=/var/lib/dcos/dcos-ui-update-service/dist/new-ui
+  - path: /etc/fluent-bit.env
+    content: |
+      FLUENT_BIT_CONFIG_FILE=/opt/mesosphere/etc/fluent-bit/fluent-bit.conf
+      FLUENT_BIT_PARSER_FILE=/opt/mesosphere/etc/fluent-bit/parsers.conf
+  - path: /etc/fluent-bit/fluent-bit.conf
+    content: |
+      [SERVICE]
+          # Flush
+          # =====
+          # Set an interval of seconds before to flush records to a destination
+          Flush        5
+          # Daemon
+          # ======
+          # Instruct Fluent Bit to run in foreground or background mode.
+          Daemon       Off
+          # Log_Level
+          # =========
+          # Set the verbosity level of the service, values can be:
+          #
+          # - error
+          # - warning
+          # - info
+          # - debug
+          # - trace
+          #
+          # By default 'info' is set, that means it includes 'error' and 'warning'.
+          Log_Level    info
+          # Parsers_File
+          # ============
+          # Specify an optional 'Parsers' configuration file
+          Parsers_File parsers.conf
+          Plugins_File plugins.conf
+          # HTTP Server
+          # ===========
+          # Enable/Disable the built-in HTTP Server for metrics
+          HTTP_Server  Off
+          HTTP_Listen  0.0.0.0
+          HTTP_Port    62020
+      [INPUT]
+          Name systemd
+          Tag host.*
+          # if /var/log/journal is unavailable (or journald is set to volatile)
+          # /run/log/journal is used instead, and this unit will not load.
+          Path /var/log/journal
+          Strip_Underscores On
+      [OUTPUT]
+          Name  null
```

Comment:
Perhaps add a comment that says "# Fluent Bit forwarding is disabled for now. It will be enabled as part of follow-up work."

Reply:
I added a comment that states we intentionally drop logs. I don't think we intend to add more outputs to this config, so I'll leave that promise out. For 1.13 we're going to allow users to configure outputs on their own.